packages/node_modules/@node-red/nodes/locales/en-US/messages.json

```
@@ -350,12 +350,37 @@
             "retain": "Retain",
             "clientid": "Client ID",
             "port": "Port",
-            "keepalive": "Keep alive time (s)",
+            "keepalive": "Keepalive time(s)",
             "cleansession": "Use clean session",
             "use-tls": "Enable secure (SSL/TLS) connection",
             "tls-config":"TLS Configuration",
             "verify-server-cert":"Verify server certificate",
-            "compatmode": "Use legacy MQTT 3.1 support"
+            "compatmode": "Use legacy MQTT 3.1 support",
+            "userProperties": "User Properties",
+            "subscriptionIdentifier": "Subscription ID",
+            "flags": "Flags",
+            "nl": "No Local",
+            "rap": "Retain as Published",
```

We can make these options more understandable - they are bit too jargony and assume you know MQTTv5.

```suggestion
            "nl": "Do not receive messages published by this client",
            "rap": "Keep retain flag of original publish",
```